### PR TITLE
Apply for Review for Issue #833

### LINKED ISSUE #883: Improve error messages
At a number of places, we show to user "error occurred" in toast messages.
This isn't helpful.

Change this message to appropriate error message in these places:
https://github.com/Swati4star/Images-to-PDF/search?q=error_occurred&unscoped_q=error_occurred

### PR DESCRIPTION
**Pull request by jayvishaalj**
# Description

The error that are shown in the snack bar were made detailed for the user to understand what actually happened.

Fixes #883 

## Type of change
Just put an x in the [] which are valid.
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [x] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

# Checklist:
- [x ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -130,6 +130,8 @@
     <string name="excel_tv_view_text">Excel file selected: </string>
     <string name="select_excel_file">Select Excel file</string>
 
+    <string name="error_path_not_found">Error Path is empty. Please select a file</string>
```
Comment (Swati4star) [inline app/src/main/res/values/strings.xml:133]: This error is when we are able to find the file path. Can you change the message here to

"Sorry, we are unable to access the file at this location."

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -146,6 +148,7 @@
     <string name="snackbar_imagecropped">Images cropped</string>
     <string name="cropImage_activityTitle">Crop Image</string>
     <string name="snackbar_pdfCreated">PDF created!</string>
+    <string name="error_uri_not_found">No file found, Please Select an image and try again</string>
```
Comment (Swati4star) [inline app/src/main/res/values/strings.xml:151]: "Sorry, We are unable to access the cropped image. Please try again"

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -518,6 +521,10 @@
     <string name="zip_to_pdf">ZIP to PDF</string>
     <string name="convert_to_pdf">convert to PDF</string>
     <string name="error_no_image_in_zip">ZIP file contains no image</string>
+    <string name="error_open_file">error occurred while opening the file, Please try reopening </string>
+
+    <!--text to pdf-->
+    <string name="error_pdf_not_created">Sorry, pdf was not created</string>
```
Comment (Swati4star) [inline app/src/main/res/values/strings.xml:527]: "Sorry, PDF was not created. Please try again."

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -518,6 +521,10 @@
     <string name="zip_to_pdf">ZIP to PDF</string>
     <string name="convert_to_pdf">convert to PDF</string>
     <string name="error_no_image_in_zip">ZIP file contains no image</string>
+    <string name="error_open_file">error occurred while opening the file, Please try reopening </string>
```
Comment (Swati4star) [inline app/src/main/res/values/strings.xml:524]: "An error occurred while opening the file. Please try reopening"

**Comment by jayvishaalj** [comment]
Hi mam,
            I have made the changes you asked for. I will push it now and
create a pull request.

Thank you
Jay Vishaal J
3rd year CSE
SSN  College of Engineering

**Comment by jayvishaalj** [comment]
I have made the changes. 👍

**Comment by jayvishaalj** [comment]
@Swati4star Are there any more changes to be made for this issue?

**Pull request merged**